packaging/ios/build.sh

```diff
@@ -72,16 +91,14 @@ fi
 
 pushd "$SUBSURFACE_SOURCE"/packaging/ios
 # create Info.plist with the correct versions
+# shellcheck disable=SC2002
 cat Info.plist.in | sed "s/@MOBILE_VERSION@/$MOBILEVERSION/;s/@CANONICAL_VERSION@/$CANONICALVERSION/;s/@PRODUCT_BUNDLE_IDENTIFIER@/$BUNDLE/" > Info.plist
 
 popd
 if [ "$versionOnly" = "1" ] ; then
 	exit 0
 fi
 
-# Build Subsurface-mobile by default
-SUBSURFACE_MOBILE=1
```

Comment:
Could this be screwing with qmake's architecture selection?

Reply:
Entirely possible.
I was so focused on getting the "real" build working that I may have removed things we need for the simulator build.

Reply:
Shall I have a look at what this does to the android build?

Reply:
This is just the shell script doing the iOS build - so it shouldn't affect the Android build at all.

I am making good progress on fixing this.

I now have a Simulator build completing again -- only the Device build fails. So I guess it's incremental improvements :)

Seriously, I gotta go to bed as my work day tomorrow starts fairly early, but I think I'll get this figured out fairly quickly.